.github/workflows/lts-release.yml

```diff
@@ -56,3 +56,42 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           content-path: ./lts/build
+
+  update_lts_table:
+    name: Automatically update the LTS table
+    runs-on: ubuntu-latest
+    needs: [lts_needed, create_lts]
+    if: ${{!needs.lts_needed.outputs.no_release}}
+    steps:
+      - name: Get Latest LTS Releases and Update the LTS config file
+        run: |
+          cat > lts.config.json <<< $(jq -r '[nth(0,1,2,3,4,5,6,7,8,9,10,11;.[] | select(.name | match(".+LTS")) | .name | rtrimstr("-LTS"))] | unique | reverse' <<< $(curl -Ls -H "Accept: application/vnd.github+json" -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/Bandwidth/api-docs/releases))
+        env:
+          TOKEN: ${{ secrets.DX_GITHUB_TOKEN }}
+
+      - name: Create Pull Request
+        id: lts-pr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.DX_GITHUB_TOKEN }}
+          commit-message: Updating LTS Versions
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          branch: Auto-Update-LTS
+          delete-branch: true
+          title: 'Updating LTS Versions'
+          body: |
+            - Auto-generated for LTS Version Update Workflow
+          labels: |
+            automated pr
+          draft: false
+        env: 
+          GITHUB_TOKEN: ${{ secrets.DX_GITHUB_TOKEN }}     
+
+      - name: Enable Pull Request Automerge
+        if: steps.lts-pr.outputs.pull-request-operation == 'created'
+        uses: peter-evans/enable-pull-request-automerge@v2
```

Comment:
It all depends on how much time we want to dedicate to something like that. It is definitely out of the scope of this card, though. We not only need an action like this but technically we need another one too that will automatically approve the PR since we need two approvals per our branch controls. So it may be the case that for the moment the better option is to just ignore the auto-merging part anyway until we get the approval side figured out (to automate we would need two other tokens so those different 'users' can do the approval)